app/controllers/application_controller.rb

```diff
@@ -1,6 +1,7 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :set_cart
+  helper_method :set_cart
 
```

Comment:
Why are we using a helper method here? If the before_action is setting the cart in the Application Controller, we shouldn't need a helper method. The only additional ability this offers us is to use that method in the view, which we shouldn't need because we have the before_action in the controller.